src/content/pages/index.md

```diff
@@ -2,32 +2,148 @@
 template: home-page
 path: /
 hero:
-  title: Threshold powers user sovereignty on the public blockchain.
-  body: Access cryptographic tools that ensure full control over your digital assets.
+  title: Full control over your digital assets
+  body: Use threshold cryptographic tools that power user sovereignty on the
+    blockchain.
   ctaButtons:
-    - label: Start Staking
+    - label: Launch Dapp
       url: https://dashboard.threshold.network
-    - label: Read the Blog
-      url: https://blog.threshold.network/
-summary:
-  title: Why Threshold Network?
-  body: Threshold Network is the first ever on-chain merge between two existing
-    networks. <a href="https://blog.threshold.network/" target="_blank"
-    rel="noopener noreferrer" >Learn more</a>
-  cards:
+    - label: Join Discord
+      url: https://discord.gg/VafbhzJ3vW
+stakerRole:
+  title: Threshold Staking
+  description:
+    Stakers escrow T tokens to run a node on the Threshold Network and
+    earn rewards.
+  image: /images/staking-threshold.png
+  buttons:
+    - label: Start Staking
+      url: https://dashboard.threshold.network/overview/network
+      variant: EXTERNAL_SOLID
+    - variant: INTERNAL_OUTLINE
+      label: Learn More
+      url: /earn/staker
+  rowReverse: false
+  bgColor: "#858585"
+lpRole:
+  title: Provide Liquidity
+  description: Liquidity providers earn yields by depositing assets into liquidity pools.
+  image: /images/liquidity.png
+  buttons:
+    - label: Learn More
+      url: /earn/liquidity-provider
+      variant: INTERNAL_SOLID
+  rowReverse: true
+  bgColor: "#858585"
```

Comment:
I'm going to  ask the designers if we can use regular hex colors for these. I think the alpha colors are not great because they're just a opacity filter over the background. In any case, these sections have editable bg colors so I think it's okay to leave it up to the CMS users to define them at the end.

Using blackAlpha doesn't look right.

wdyt?